### ISSUE TITLE
Sex for clinical use

### ISSUE THREAD

**Issue by kruzikh** [Xt-EHR/xt-ehr-common#32]
Marcello: sex for clinical use is based on genetic disposition of the patient. It is not changing over time! As such there i no reason to remove it from patient model.

**Comment by danka74** [comment]
See https://confluence.hl7.org/pages/viewpage.action?pageId=42993178
Due to the fractal nature of "biological sex" (is it reproductive organs, chromosomes, hormones, or etc. as per the WHO definition?) I would still propose to move this to observation results.

Also, not changing cannot be a factor, e.g. full DNA (which is not there) does not change while name and address (which is there) will.

**Comment by rlindstrm** [comment]
"Sex for clinical use" is not the same as "biological sex" or "sex at birth". And like stated by Daniel, "sex at birth" may not change over time, but there are various ways to determine the sex and for intersex people the results are dependent on the assessment method.  
"Sex for clinical use" must be defined carefully. You cannot treat a transgender person according to their sex at birth. Clinical treatment is specific to every step of the transition. This means, the value would definitely change over time repeatedly.

In FHIR, there is an [extension ](https://build.fhir.org/ig/HL7/fhir-extensions/StructureDefinition-patient-sexParameterForClinicalUse.html#profile) and a value set for sex for clinical use, it's marked experimental and the valueset itself doesn't add anything to AdministrativeGender.  It's a complex extension with period and comment and a reference to any other resource with actual information (like Observation).

If we really want to have this element in Patient, we should follow this complex extension path, not rely on a single element.

**Comment by kruzikh** [comment]
Agree, I removed sex for clinical use from patient logical model. Sex for clinical use is now modeled as supporting information in the lab report body and imaging report body. It is to be decided where to model it in the scope of PS and HDR!

**Comment by mikatuomainen** [comment]
Maybe late, but here is different aspects to sex and gender representation ala HL7: 
https://build.fhir.org/ig/HL7/fhir-gender-harmony/index.html

Especially FHIR: 
- https://build.fhir.org/ig/HL7/fhir-gender-harmony/fhirgenderharmony.html
- https://build.fhir.org/ig/HL7/fhir-gender-harmony/fhirdesignbackground.html

**Comment by io-Nic** [comment]
Decided on 30-09-2025:
Error in the cardinality: change cardinality from 0..* to **0..1.**
Also, we will wait for stakeholder consultation for possible comments regarding this subject.
